includes/admin/settings/class-settings-display.php

```diff
@@ -87,6 +87,17 @@ public function get_settings() {
 								'optional' => __( 'Optional', 'give' ),
 							],
 						],
+						[
+							'name'    => __( 'Last Name Required', 'give' ),
+							'desc'    => __( 'Do you want to force the Last Name field to be required?', 'give' ),
```

Comment:
"Do you want to require donor last names/surnames? By default, only the first name field on a donation is required. This setting is also configurable on a per-form basis."

Reply:
Resolved 865c49d6